<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferTab</class>
 <widget class="QWidget" name="TransferTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>890</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="filterArea" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="4">
       <widget class="QLabel" name="lblFilterText">
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="lblFilterEndDate">
        <property name="text">
         <string>End Date:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDateEdit" name="filterEndDate">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2038</year>
          <month>12</month>
          <day>31</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>d. MMMM yyyy</string>
        </property>
        <property name="date">
         <date>
          <year>2038</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="filterText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Use column:value to filter specfic columns, also try checked, unchecked, internal, ...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QDateEdit" name="filterStartDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1970</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>d. MMMM yyyy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblFilterStartDate">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filterMonthLinks"/>
   </item>
   <item>
    <widget class="QWidget" name="statsArea" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="statsRevenues">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblIncome">
        <property name="text">
         <string>Income:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblExpenses">
        <property name="text">
         <string>Expenses:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="statsProfit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="lblProfit">
        <property name="text">
         <string>Profit:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="statsExpenses">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblInternal">
        <property name="text">
         <string>Internal:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="statsInternal">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="tableArea">
     <item>
      <widget class="QTableView" name="transfers"/>
     </item>
     <item>
      <widget class="QWidget" name="detailsArea" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lblFrom">
          <property name="text">
           <string>From:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="detailsFrom">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblReference">
          <property name="text">
           <string>Reference:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="detailsReference">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblAccountTags">
          <property name="text">
           <string>Account Tags:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="detailsAccountTags">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblNote">
          <property name="text">
           <string>Note:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="detailsNote">
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblTags">
          <property name="text">
           <string>Tags:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TagEdit" name="detailsTags">
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="checkSelected">
          <property name="text">
           <string>Check Selected</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>src/ui/TagEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterStartDate</tabstop>
  <tabstop>filterEndDate</tabstop>
  <tabstop>filterText</tabstop>
  <tabstop>transfers</tabstop>
  <tabstop>detailsNote</tabstop>
  <tabstop>detailsTags</tabstop>
  <tabstop>checkSelected</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
